services/secure-gateway/main.go

```diff
@@ -70,7 +70,9 @@ func newApp(config *appConfig.Configuration) http.Handler {
 	authenticationMiddleware := security.AuthenticationMiddleware(&config.Security, auditor)
 	authorizationMiddleware := security.AuthorizationMiddleware(enforcer, auditor)
 
-	middleware := negroni.New(negroni.NewRecovery())
+	recovery := negroni.NewRecovery()
+	recovery.PrintStack = false
```

Comment:
There is an open issue about this
https://github.com/urfave/negroni/issues/214
So I think it will be redundant.
Good for now.